docs/rich_presence/Certified_Devices.md

```diff
@@ -0,0 +1,158 @@
+# Certified Devices
+
+Baked into Discord's Rich Presence functionality is the ability for hardware manufacturers to tell us a little more about the certified devices that are plugged into a user's computer. Unfortunately, no, you can't show that a user's PUBG Chicken Dinner was all thanks to the amazing TotallyRealHardware RGB Mouse and Keyboard Set Extraordinaire™, but you _can_ give them an amazing experience using your hardware with Discord!
+
+## How's it work?
+
+I'm glad you asked!
+
+1. [Create an application](https://discordapp.com/developers/applications/me) for your hardware vendor—save the Client ID!
+2. Talk to Discord via one simple HTTP call (or RPC, if you're already whitelisted for it)
```

Comment:
WebSocket use does not require whitelisting for this. this is also all RPC, so this should be `or WebSocket`. I'm curious if we should document IPC at some point too, since technically we support that as well.